configs/config.ini

```diff
@@ -75,14 +75,6 @@ keep_volume_storage = False
 # b/kb/mb/gb - a multiple of the unit byte for glance image info transfer
```

Comment:
i think, we do not need this file, we can generate it at any time: `oslo-config-generator --namespace cloudferry > conf.ini`